### PR TITLE
changed link

### PR DESCRIPTION
**Pull request by sanjar1311**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -98,8 +98,9 @@ img {
 }
 
 .request-link {
-  background-color: linear-gradient(135deg, #33D35E 0%, #2AB6D9 99.58%);
```
Comment (sanjar1311) [inline css/styles.css:101]: bu backgroun img boladi brat

File: css/styles.css [hunk 2]
```diff
@@ -112,9 +113,11 @@ img {
   line-height: 28px;
   padding: 8px 36px 8px 33px;
   display: block; 
+  
 }
 .request:hover {
-  background-color:  #75e498;
```
Comment (sanjar1311) [inline css/styles.css:117]: bu keremasti

**Review by sanjar1311 — COMMENTED** [review]
zor

File: css/styles.css (hunk 2 above)
Comment (iskandarov101) [reply css/styles.css:117]: boldi gap yoq bratan

File: css/styles.css (hunk 1 above)
Comment (iskandarov101) [reply css/styles.css:101]: yoooooolllllllgooooooooonnnnnnnnnnn

**Comment by sanjar1311** [comment]
nma gaaaap


ср, 14 окт. 2020 г. в 16:25, iskandarov101 <notifications@github.com>:

**Pull request merged**